packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.vue

```diff
@@ -59,9 +67,28 @@ export default {
       type: Boolean,
       default: false,
     },
+    /** Minimum allowed quantity */
+    min: {
+      type: Number,
+      required: false,
```

Comment:
If a prop has a default value it automatically means that it is not required.
```suggestion
```

Reply:
Yeah, for these I added required: false for verbosity/clarity. I tend to lean towards more understandable code versus minimal code, because I usually am trying to write code for developers who aren't as familiar with the technology. But I'll remove these as it makes sense to keep it minimalistic in a lib like this.